packages/modeling/src/operations/booleans/retessellate.js

```diff
@@ -5,11 +5,21 @@ const poly3 = require('../../geometries/poly3')
 
 const reTesselateCoplanarPolygons = require('./reTesselateCoplanarPolygons')
 
+// Normals are directional vectors of length 0 to 1.0
```

Comment:
Is this true in JSCAD? 

Mathematically, normals have a length of exactly 1, although each dimension is between 0 and 1.

Reply:
Good point.

Reply:
Updated comment.